README.md

```diff
@@ -1,2 +1,100 @@
-# Getting-started-with-Syncfusion-Grid-component-in-Streamlit-app
-This repository contains the Syncfusion Grid component example using Streamlit application.
+# Getting started with Syncfusion Grid component with Streamlit Python Framework
+
+This article provides a step-by-step guide for setting up a [Streamlit](https://streamlit.io/) project with a Python environment and integrating the Syncfusion Grid components.
+
+The `Streamlit` is a free and open-source framework that enables quick development and sharing of visually appealing web applications for machine learning and data science.
+
+## Set up the Streamlit project environment
+
+A recommended approach for beginning with Streamlit framework is to create a Python environment with [venv](https://docs.python.org/3/library/venv.html). To create a new Python environment, use the following command.
+
+```bash
+python -m venv my-project
+```
+
+Using the above command will create a Python virtual environment.
+
+Upon completing the aforementioned step to create **my-project**, run the following command to install Streamlit:
+
+```bash
+cd my-project
+pip install streamlit
+```
+
+Now that **my-project** is ready to run with default settings, let's add Syncfusion Grid components to the project.
+
+## Add Syncfusion Grid package
+
+Syncfusion Grid component package is available at [pypi.org](https://pypi.org/project/ej2-streamlit-grids/). To use the Syncfusion Grid component in the project, install the corresponding pip package.
+
+```sh
+pip install ej2-streamlit-grids
+```
+
+## Add Syncfusion Grid Themes
```

Comment:
remove this heading and add it into 4 th point

Reply:
Heading was removed and added as fourth point.